src/scripts/browserDeviceProfile.js

```diff
@@ -328,7 +362,7 @@ import browser from './browser';
                                     || videoTestElement.canPlayType('video/mp4; codecs="avc1.640029, mp3"').replace(/no/, '');
 
         // Not sure how to test for this
-        const supportsMp2VideoAudio = browser.edgeUwp || browser.tizen || browser.web0s;
+        const supportsMp2VideoAudio = browser.edgeUwp || browser.tizen || browser.web0s || browser.chrome || browser.edgeChromium;
```

Comment:
It'd be good to document why you're adding this and how to test it.

Reply:
Can be useful for TV or DVD rips, both use MPEG 1 Layer 2 heavily.
`ffmpeg -i source.mp4 -c:a mp2 output.mp4`